features/issue.feature

```diff
@@ -80,6 +80,34 @@ Feature: hub issue
     """
     When I successfully run `hub issue -d 2016-08-18T09:11:32Z`
 
+  Scenario: Fetch issues
+    Given the GitHub API server:
+    """
+    get('/repos/github/hub/issues') {
+      assert :sort => "comments"
+      assert :direction => "asc"
+
+      json [
```

Comment:
I prefer returning an array to make sure that the sort is done by GitHub and that the order is kept exactly as such by hub.